tests/topotests/multicast_pim6_dual_stack/test_multicast_pim6_dual_stack.py

```diff
@@ -0,0 +1,1936 @@
+            "iif": topo["routers"]["r1"]["links"]["r4"]["interface"],
+            "oil": topo["routers"]["r1"]["links"]["i1"]["interface"],
+        },
+        {
+            "dut": "r4",
+            "src_address": source_v4,
+            "iif": topo["routers"]["r4"]["links"]["i7"]["interface"],
+            "oil": topo["routers"]["r4"]["links"]["r1"]["interface"],
+        },
+    ]
+
+    for data in input_dict_v4:
+        result = verify_mroutes(
+            tgen,
+            data["dut"],
+            data["src_address"],
+            IGMP_JOIN_RANGE_1,
+            data["iif"],
+            data["oil"],
+        )
+        assert result is True, "Testcase {} : Failed Error: {}".format(tc_name, result)
+
+        result = verify_upstream_iif(
+            tgen, data["dut"], data["iif"], data["src_address"], IGMP_JOIN_RANGE_1
+        )
+        assert result is True, "Testcase {} : Failed Error: {}".format(tc_name, result)
+
+    input_dict_v6 = [
+        {
+            "dut": "r4",
+            "src_address": source_v6,
+            "iif": topo["routers"]["r4"]["links"]["i7"]["interface"],
+            "oil": topo["routers"]["r4"]["links"]["r1"]["interface"],
+        },
+        {
+            "dut": "r1",
+            "src_address": source_v6,
+            "iif": topo["routers"]["r1"]["links"]["r4"]["interface"],
+            "oil": topo["routers"]["r1"]["links"]["i1"]["interface"],
+        },
+    ]
+
+    step(
+        "'show ipv6 pim upstream' and 'show ipv6 pim upstream-rpf' showing"
+        " correct OIL and IIF on all the nodes"
+    )
+
+    for data in input_dict_v6:
+        result = verify_mroutes(
+            tgen,
+            data["dut"],
+            data["src_address"],
+            MLD_JOIN_RANGE_1,
+            data["iif"],
+            data["oil"],
+        )
+        assert result is True, "Testcase {} : Failed Error: {}".format(tc_name, result)
+
+        result = verify_upstream_iif(
+            tgen, data["dut"], data["iif"], data["src_address"], MLD_JOIN_RANGE_1
+        )
+        assert result is True, "Testcase {} : Failed Error: {}".format(tc_name, result)
+
+    step(
+        "IPv4 and IPv6 traffic updated per (S,G)"
+        "show ipv6 multicast count ,show ip multicast count"
+    )
+
+    step("verify ipv6 multicast traffic on all the grps")
+    result = verify_sg_traffic(tgen, "r1", MLD_JOIN_RANGE_1, source_v6, "ipv6")
+    assert result is True, "Testcase {} : Failed Error: {}".format(tc_name, result)
+
+    step("verify ipv4 multicast traffic on all the grps")
+    result = verify_sg_traffic(tgen, "r1", IGMP_JOIN_RANGE_1, source_v4, "ipv4")
+    assert result is True, "Testcase {} : Failed Error: {}".format(tc_name, result)
+
+    step(
+        "Verify IP pimv6 join and IPv6 pimv6 join to FRR1 and check join is sent toward FHR"
+        "Join counts is increamented for both IPv4 and IPv6"
+        'verify using "show ip pimv6 intrface traffic" "show ipv6 pimv6 interface traffic"'
```

Comment:
please change intrface --> interface. Typo.

Reply:
Done, Thank you